imagetagger/imagetagger/settings.py

```diff
@@ -0,0 +1,230 @@
+
+    PROBLEMS_URL = 'https://github.com/bit-bots/imagetagger/issues'
+    PROBLEMS_TEXT = ''
+
+    LOGIN_URL = '/user/login/'
+    LOGIN_REDIRECT_URL = '/images/'
+
+    # Flash Messages
+    # https://docs.djangoproject.com/en/3.1/ref/contrib/messages/
+
+    MESSAGE_STORAGE = 'django.contrib.messages.storage.session.SessionStorage'
+    MESSAGE_TAGS = {
+        messages.INFO: 'info',
+        messages.ERROR: 'danger',
+        messages.WARNING: 'warning',
+        messages.SUCCESS: 'success',
+    }
+
+    # Sets the default expire time for new messages in days
+    DEFAULT_EXPIRE_TIME = 7
+
+    # Sets the default number of messages per page
+    MESSAGES_PER_PAGE = 10
+
+    # Static files (CSS, JavaScript, Images)
+    # https://docs.djangoproject.com/en/1.10/howto/static-files/
+
+    STATIC_URL = '/static/'
+
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+    ########################################################################
+    #
+    # Computed properties and hooks
+    #
+    ########################################################################
+    @property
+    def DATABASES(self):
+        """https://docs.djangoproject.com/en/1.10/ref/settings/#databases"""
+        return {
+            'default': {
+                'ENGINE': 'django.db.backends.postgresql_psycopg2',
+                'HOST': self.DB_HOST,
+                'PORT': self.DB_PORT,
+                'NAME': self.DB_NAME,
+                'USER': self.DB_USER,
+                'PASSWORD': self.DB_PASSWORD
+            }
+        }
+
+    @classmethod
+    def post_setup(cls):
+        super().post_setup()
+
+        if cls.SENTRY_REPORTING_ENABLED:
+            try:
+                import sentry_sdk
+                from sentry_sdk.integrations.django import DjangoIntegration
+                sentry_sdk.init(
+                    dsn=cls.SENTRY_DSN,
+                    integrations=[DjangoIntegration()],
+                    # If you wish to associate users to errors you may enable sending PII data.
+                    send_default_pii=cls.SENTRY_SEND_DEFAULT_PII
+                )
+            except ImportError:
+                raise ImproperlyConfigured("Could not import sentry although the server is configured to use it")
+
+    ########################################################################
+    #
+    # User-adaptable settings
+    #
+    ########################################################################
+    DEBUG = values.BooleanValue(environ_prefix='IT', default=False)
+    SECRET_KEY = values.SecretValue(environ_prefix='IT')
+    DB_HOST = values.Value(environ_prefix='IT', environ_required=True)
+    DB_PORT = values.PositiveIntegerValue(environ_prefix='IT', default=5432)
+    DB_NAME = values.Value(environ_prefix='IT', default='imagetagger')
+    DB_USER = values.Value(environ_prefix='IT', default=DB_NAME)
+    DB_PASSWORD = values.Value(environ_prefix='IT', default=DB_USER)
```

Comment:
`default=DB_USER` does not really make sense.

Reply:
It made sense because in a dev environment that is my setup. I have made this more explicit by not providing a default here and then overriding it in the dev config.